brainstormforce/ultimate-addons-for-gutenberg
# Remove old asset files on asset's version update

### Description
Remove old asset files on asset's version update

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
- Regenerate assets should load the different file on refresh
- Delete files from the upload directory
- Do changes and Update the post.
- Create a page in master and switch to next release

### Checklist:
